.changeset/bright-vans-add.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/preview-middleware': patch
+---
+
+Add integration test
```

Comment:
I am not sure, if we actually need a changeset. We are not changing anything in the functionality of the package

Reply:
in UI5 LA if there is any changes in any package, we include them in change set. I think we do same strategies in open-ux tools. Removing it  may cause pipeline checks fails.

Reply:
I think that we can safely remove the changeset in this case. @devinea Is my assumption correct?

Reply:
If the new module is private then a changeset is not needed.

Reply:
On second thoughts, to write the change log properly it is best to include the changeset. 
Publishing to npm is not applicable in this case.

Reply:
OK